crates/topos-test-sdk/src/tce/gatekeeper.rs

```diff
@@ -9,9 +9,11 @@ use topos_tce_gatekeeper::{Client, GatekeeperError};
 
 pub async fn create_gatekeeper<P: Into<PeerId>>(
     peer_id: P,
+    peer_list: Vec<PeerId>,
```

Comment:
Shouldn't this be generic over `P` too? `peerlist: Vec<P>` (or remove the generic altogether, which would be even better if possible)